homeassistant/components/hardware/websocket_api.py

```diff
@@ -45,3 +72,48 @@ async def ws_info(
                 hardware_info.append(asdict(platform.async_info(hass)))
 
     connection.send_result(msg["id"], {"hardware": hardware_info})
+
+
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "hardware/subscribe_system_status",
+    }
+)
+@websocket_api.async_response
+async def ws_subscribe_system_status(hass, connection, msg):
+    """Subscribe to system status updates."""
+
+    system_status: SystemStatus = hass.data[DOMAIN]["system_status"]
+
+    @callback
+    def async_update_status(now: datetime) -> None:
+        cpu_percentage = round(system_status.psutil.cpu_percent(interval=None))
+        virtual_memory = system_status.psutil.virtual_memory()
+        json_msg = {
+            "cpu_percentage": cpu_percentage,
+            "memory_use_percent": virtual_memory.percent,
+            "memory_use": round(
+                (virtual_memory.total - virtual_memory.available) / 1024**2, 1
+            ),
+            "memory_free": round(virtual_memory.available / 1024**2, 1),
+        }
```

Comment:
```suggestion
        json_msg = {
            "cpu_percent": cpu_percentage,
            "memory_used_percent": virtual_memory.percent,
            "memory_used_mb": round(
                (virtual_memory.total - virtual_memory.available) / 1024**2, 1
            ),
            "memory_free_mb": round(virtual_memory.available / 1024**2, 1),
        }
```

Better place for the comment